core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -683,4 +683,34 @@
         ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponce);
     }
+
+    /**
+     * Method interceptor exception {@link RegionNotFoundException}.
+     *
+     * @param ex         Exception which should be intercepted.
+     * @param webRequest contain detail about occur exception.
+     * @return ResponseEntity which contain http status and body with message of
+     *         exception.
+     */
+    @ExceptionHandler({RegionNotFoundException.class})
+    public final ResponseEntity<Object> handleRegionNotFoundException(RegionNotFoundException ex,
+        WebRequest webRequest) {
+        ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponce);
+    }
+
+    /**
+     * Method interceptor exception {@link LocationAlreadyCreatedException}.
+     *
+     * @param ex         Exception which should be intercepted.
+     * @param webRequest contain detail about occur exception.
+     * @return ResponseEntity which contain http status and body with message of
+     *         exception.
+     */
+    @ExceptionHandler({LocationAlreadyCreatedException.class})
+    public final ResponseEntity<Object> handleLocationAlreadyCreatedException(LocationAlreadyCreatedException ex,
+        WebRequest webRequest) {
+        ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponce);
```

Comment:
You don't need to have multiple ExceptionHandlers that return the same BadRequest. You can define one handler and keep list of Exceptions in @ ExceptionHandler annotation